api.ts

```diff
@@ -1285,8 +1287,12 @@ export interface SyncTableOptions<
    * For example, if your identity name was 'Product', another formula or sync could return
    * shell objects that reference rows in this table, so long as they contain the id
    * of the object, and the schema is declared as `{identity: {name: 'Products'}}`.
+   *
+   * The only time you don't need to specify this is if your schema already sets it's own
+   * `identity.name`. Technically, this value is only the default identity name for your table's
+   * schema(s).
    */
-  identityName: string;
+  identityName?: string;
```

Comment:
Per discussion, sounds like we'll leave this required. I'll wait to review further until you've updated!